enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +272,35 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Console's frontend will use `/locales/resource.json?lng={{lng}}&ns={{ns}}`
+as i18next loadPath.
+
+#### Dynamic Plugins
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
+per language, called `default.json`.
+All dynamic plugins are expected to use `plugin:` namespace prefix, e.g.
+`plugin:dev-console` or `plugin:kubevirt-plugin`.
+The request for the dynamic plugin locales will be proxied by console to plugin's `Pod`,
+using it's `Service`.
+Example request path for `console-demo-plugin` locales will look like:
+`/locales/resource.json?lng=en&ns=plugin:console-demo-plugin'`.
+This request will be proxied to:
+`<service-name>.<service-namespace>.svc.cluster.local:<service-port>/locales/{language}/default.json`
+
+#### Static Plugins
```

Comment:
I'd leave the section on static plugins out of the enhancement proposal. We're working to migrate away from those, and they're loaded the same way as core console translations are.